### PR TITLE
Update install_library.zh-cn.md

### PR DESCRIPTION
**Pull request by obbimasheep**
Text translations: English to Chinese.
Clarified the differences between "Driver Install"  and "Library Install",in now version Chinese translation-text all means  "Driver Install".

### Description
Chinese language translation

### CODE REVIEW

**Comment by diemol** [comment]
@obbimasheep could you please sign the CLA so we can review this?

File: website_and_docs/content/documentation/webdriver/getting_started/install_library.zh-cn.md [hunk 1]
```diff
@@ -14,59 +14,59 @@ aliases: [
 首先，您需要为自动化项目安装 Selenium 绑定库。
 库的安装过程取决于您选择使用的语言。
 
-## Requirements by language
+## 请求对应的程序语言
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/getting_started/install_library.zh-cn.md:17]: Where is 请求？

**Review by alaahong — APPROVED** [review]
Almost are fine. I can submit a revised version after merged.

**Comment by obbimasheep** [comment]
> @obbimasheep could you please sign the CLA so we can review this?
I'm a newbie for github ,  signed the CLA  


> @obbimasheep could you please sign the CLA so we can review this?

A newbie for github , the CLA signing is done , I hope it works.

**Review by diemol — APPROVED** [review]
Thank you, @obbimasheep!

Merging as @alaahong will follow with another PR.

**Pull request merged**